packages/playground/src/weblets/micro_vm.vue

```diff
@@ -91,7 +91,7 @@
             :value="envs[index].key"
             :rules="[
               validators.required('Key name is required.'),
-              key => validators.isAlpha('Key must start with alphabet char.')(key[0]),
+              (key: string) => validators.isAlpha('Key must start with alphabet char.')(key[0]),
```

Comment:
```suggestion
              (key: string) => validators.isAlpha('Key must start with alphabetical character.')(key[0]),
```



please fix all other similar validation messages accordingly